v20.1/timestamp.md

```diff
@@ -99,15 +101,15 @@ A `TIMESTAMP`/`TIMESTAMPTZ` column supports values up to 12 bytes in width, but
 ~~~
 
 ~~~
+  a |             b
 +---+---------------------------+
-| a |             b             |
-+---+---------------------------+
-| 1 | 2016-03-26 15:10:10+00:00 |
-| 2 | 2016-03-26 00:00:00+00:00 |
-+---+---------------------------+
-# Note that the first timestamp is UTC-05:00, which is the equivalent of EST.
+  1 | 2016-03-26 15:10:10+00:00
+  2 | 2016-03-26 00:00:00+00:00
+(2 rows)
 ~~~
 
+Note that the first timestamp is UTC-05:00, which is the equivalent of EST.
```

Comment:
i would mention this is because the context timezone is in UTC. if we moved this to +1, this would display 16:10